src/Components/OATGraphViewer/OATGraphViewer.tsx

```diff
@@ -126,6 +128,18 @@ const OATGraphViewer = ({ state, dispatch }: OATGraphProps) => {
         }
     }, [model]);
 
+    useEffect(() => {
+        // Reload elements on project change
+        if (reloadProject) {
+            setElements(getStoredElements());
+
+            dispatch({
+                type: SET_OAT_RELOAD_PROJECT,
+                payload: false
+            });
+        }
+    }, [reloadProject]);
```

Comment:
I wonder if setting a flag is the right approach here. Perhaps we should be pull the elements state up into the editor page? So we can set it from there on project change?